libfreerdp/core/nego.c

```diff
@@ -1471,8 +1481,8 @@ BOOL nego_send_negotiation_response(rdpNego* nego)
 		if (freerdp_settings_get_bool(settings, FreeRDP_SupportGraphicsPipeline))
 			flags |= DYNVC_GFX_PROTOCOL_SUPPORTED;
 
-		if (freerdp_settings_get_bool(settings, FreeRDP_RestrictedAdminModeRequired))
-			flags |= RESTRICTED_ADMIN_MODE_SUPPORTED;
+		/* Server always supports RestrictedAdmin */
```

Comment:
not sure this is a good idea.
we use the `rdpSettings` to have a way to enable/disable things and set the default values to what is expected.
(allows easier testing of *impossible* situation handling)

Reply:
That's okay, but in this case we need to add a new `FreeRDP_RestrictedAdminSupported` flag and some extra logic

Reply:
I've changed the PR to add a new settings to handle this behavior.